providers/apis/dydx/multi_market_map_fetcher.go

```diff
@@ -1,186 +0,0 @@
-package dydx
```

Comment:
Redundant as the research json already has mainnet markets. Moving forward deployments are going to be:

1. dYdX staging (w/o VPN) pointing to mainnet
2. dYdX staging (w VPN) pointing to mainnet
3. dYdX research (w VPN)
4. dYdX research (w/o VPN)
5. dYdX research coinmarketcap